pandas/core/indexes/range.py

```diff
@@ -146,21 +146,18 @@ def from_range(cls, data, name=None, dtype=None):
         return cls._simple_new(data, dtype=dtype, name=name)
 
     @classmethod
-    def _simple_new(cls, values, name=None, dtype=None, **kwargs):
+    def _simple_new(cls, values, name=None, dtype=None):
         result = object.__new__(cls)
 
         # handle passed None, non-integers
         if values is None:
             # empty
             values = range(0, 0, 1)
         elif not isinstance(values, range):
-            return Index(values, dtype=dtype, name=name, **kwargs)
+            return Index(values, dtype=dtype, name=name)
```

Comment:
not for this PR, but it'd be nice tot see this branch gone; doesn't belong in _simple_new

Reply:
Yeah, I had that in my first version, but figured that is a different issue.